src/actions/filter.ts

```diff
@@ -0,0 +1,26 @@
+import {FilterTransform} from 'vega-lite/build/src/transform';
+import {ReduxAction} from './redux-action';
+
+export type FilterAction = AddRangeFilter | AddOneOfFilter | RemoveRangeFilter | RemoveOneOfFilter ;
```

Comment:
Besides Adding / Removing, you will have to support modifying range / oneof filter as well.   
(But I'm fine if the modifying parts come in another PR, which will make it easier to review.)